Makefile

```diff
@@ -4,6 +4,7 @@
 
 GO = GO111MODULE=on go
 GO_FILES ?= ./pkg/...
+SH_FILES ?= $(shell find ./scripts -name *.sh)
```

Comment:
@aocenas you might be interested in this solution

Reply:
In the end I went with this:
```
# recursive wildcard https://stackoverflow.com/a/12959764/1835885
rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
```

Not sure if important (I think that ship already sailed) but it should also work on Win as it does not use shell.